rviz_common/src/rviz_common/add_display_dialog.cpp

```diff
@@ -140,7 +140,7 @@ struct PluginGroup
 };
 
 void getPluginGroups(
-  const QMap<QString, QString> & datatype_plugins,
+  const QMultiMap<QString, QString> & datatype_plugins,
```

Comment:
If we want to avoid this API change for Foxy, we can instead try to quiet the build warnings.

Reply:
I don't think this API is public. The only thing that changed that is public is the type of the private member, no? I think this should be fine.